.palantir/revapi.yml

```diff
@@ -66,6 +66,10 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.io.DataWriter<T>::add(T)"
       justification: "Removing deprecated method"
   "1.1.0":
+    org.apache.iceberg:iceberg-api:
```

Comment:
we shouldn't introduce breaking API changes, but rather make `deletedStatisticsFilesCount()` return 0 by default

Reply:
Nice. I didn't know it was considered as breaking change because the default implementation is missing. TIL. Fixed it now.

Reply:
In general if RevAPI is complaining about something, then it's a breaking API change. We just need to be careful that we only add API breaks that we previously deprecated as defined in https://github.com/apache/iceberg/blob/master/CONTRIBUTING.md#major-version-deprecations-required

Reply:
Ack.